src/material-experimental/mdc-menu/menu.html

```diff
@@ -1 +1,16 @@
-<!-- TODO: MDC template here. -->
+<ng-template>
+  <div
+    class="mat-mdc-menu-panel mdc-menu-surface mdc-menu-surface--open"
+    [ngClass]="_classList"
+    (keydown)="_handleKeydown($event)"
+    (click)="closed.emit('click')"
+    [@transformMenu]="_panelAnimationState"
+    (@transformMenu.start)="_onAnimationStart($event)"
+    (@transformMenu.done)="_onAnimationDone($event)"
+    tabindex="-1"
+    role="menu">
+    <div class="mat-mdc-menu-content mdc-list">
+      <ng-content></ng-content>
```

Comment:
Not in this PR, but I want to see if we can avoid the "ng-content inside ng-template` pattern with the MDC components, even if it means we have to do a change on the existing components first. We can talk about it at our team meeting.

Reply:
We do support providing an `ng-template` to `mat-menu`, but it's a little bit of extra HTML and it makes timing animations slightly trickier.